maas/client/viscera/machines.py

```diff
@@ -67,11 +67,17 @@ class MachinesType(ObjectType):
         else:
             return cls._object(data)
 
-    async def get_power_parameters(
-            cls, *, system_ids: typing.Sequence[str]=[]):
+    async def get_power_parameters_for(
+            cls, *, system_ids: typing.Sequence[str]):
         """
+        Get a list of power parameters for specified systems.
+        *WARNING*: This method is considered 'alpha' and may be modified
+        in future.
+
         :param system_ids: The system IDs to get power parameters for
         """
+        if len(system_ids) == 0:
+            return {}
         data = await cls._handler.power_parameters(id=system_ids)
         return data
```

Comment:
Still not a fan of how this just returns a dictionary and not a Machine object. But it will work for now.